rollup.config.js

```diff
@@ -86,7 +88,7 @@ export default [
     plugins: [
       nodeResolve(),
       babel(getBabelOptions({useESModules: true})),
-      commonjs(commonjsOptions),
+      reactPkg && commonjs(commonjsOptions),
```

Comment:
Why do we need it?

Reply:
To make sure we will never ever add another commonjs package in our dependencies.

Reply:
but why this conditional and not for everything?

Reply:
Because we dropped commonjs for everything except react based packages.

Reply:
it just seems like we are addng there another list to maintain

Reply:
I don't think it will be changed before react will accept esm.